istioctl/cmd/proxyconfig.go

```diff
@@ -766,3 +808,18 @@ func proxyConfig() *cobra.Command {
 
 	return configCmd
 }
+
+func getPodName(podflag string) (string, string, error) {
```

Comment:
This looks great. How about moving `getPodName ` into `handlers` and utilize it for `authz`, `dashboard` and `proxy-status`?